src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,57 @@
+package duke.task;
+
+/**
+ * Represents a Todo object.
+ */
+public class Todo extends Task {
+    /**
+     * Creates a todo.
+     * @param name A string representing the name of the todo.
+     * @param isCompleted A boolean value representing whether the todo has been completed.
+     */
+    protected Todo(String name, boolean isCompleted) {
+        super(name, isCompleted);
+    }
+
+    /**
+     * Creates a new Todo.
+     * The completion status is false by default.
+     * @param name A string representing the name of the new todo.
+     * @return The new todo created.
+     */
+    public static Todo newTodo(String name) {
```

Comment:
createTodo(...)

Reply:
Thanks for your feedback on my method names! It seems I overlooked the convention of making the method names a verb. 😅